plugin.py

```diff
@@ -29,6 +33,16 @@ class LspTypescriptPlugin(NpmClientHandler):
     def minimum_node_version(cls) -> Tuple[int, int, int]:
         return (14, 16, 0)
 
+    @classmethod
+    def on_pre_start(
+        cls,
+        window: sublime.Window,
+        initiating_view: sublime.View,
+        workspace_folders: list[WorkspaceFolder],
+        configuration: ClientConfig,
+    ) -> str | None:
+        cls._support_vue_hybrid_mode(configuration)
```

Comment:
Problems:
- This will not trigger until one opens a supported file type (ts/js/..). So none of this logic will run when opening a *.vue file.
- This changes global LSP-typescript settings so the vue plugin will be enabled for every project/file once some *.vue file is opened. Doesn't seem ideal. I wouldn't trust the vue plugin enough to enable it at all times, for all projects.

Reply:
Maybe we should avoid all the magic if we can: https://github.com/sublimelsp/LSP-vue/pull/136#issuecomment-3172447162